### DISCUSSION TITLE
Is there a way to enable typescript support in the Interactive code editor?

### DISCUSSION THREAD

**Discussion by JacobWeisenburger** [facebook/docusaurus#6819] [Q&A] [answered]
Is there a way to enable typescript support in the [Interactive code editor](https://docusaurus.io/docs/next/markdown-features/code-blocks#interactive-code-editor)?

![image](https://user-images.githubusercontent.com/31667350/156445281-f16e2495-9a43-4a17-a180-738a99951d6b.png)
---
Typescript works with the regular code block.

![image](https://user-images.githubusercontent.com/31667350/156445532-17180c2a-c4fd-4bd9-b48e-a6c6ea76c4ec.png)

**Comment by slorber** [comment, marked as answer]
We don't support TypeScript atm but we may soon, thanks to @nihgwu PR https://github.com/facebook/docusaurus/pull/6589

**Comment by nhanphan2112** [comment]
👋 just want to check in on this thread...Wondering if there's plan to support TypeScript for the interactive code editor?

**Reply by slorber** [reply]
We don't support it right now but with MDX you can provide your own live editor if you want, for example try https://sandpack.codesandbox.io/
